civitai-downloader-v2/src/cli/main.py

```
@@ -8,17 +8,20 @@
 import asyncio
 import json
 import sys
+import csv
+import io
 from pathlib import Path
 from typing import List, Optional
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)